views/moderation.js

```diff
@@ -134,11 +187,16 @@ module.exports = function (cabal, modKey, db) {
           })
           if (--pending === 0) cb(null, banned)
         })
+      },
+
+      getRole: function (core, opts, cb) {
```

Comment:
Great API expansion! Do you have capacity to add these to the README docs as you go?